Makefile

```diff
@@ -39,7 +39,7 @@ build-testing:
 build-soroban-dev:
 	$(MAKE) build TAG=soroban-dev \
 		XDR_REF=v20.0.1 \
-		CORE_REF=6177299100b114aa108584053414371f38aebf53 \
+		CORE_REF=v20.0.0 \
```

Comment:
I'm the one that originally had the issue with the tags (I think @sreuland did as well), but I can build using this PR so I'm not sure what was going on... 

Do you mind updating line 38 to remove "CORE_REF" from the comment?

Reply:
Removed comments and removed rpc sha too.